src/components/Image.js

```diff
@@ -70,26 +72,41 @@ export const Image = ({
         break effect;
       }
 
-      sourceProp.uri
-        ? CacheManager.get(sourceProp.uri)
-            .getPath()
-            .then((path) => {
-              mounted && setSource({ uri: path });
-            })
-            .catch((err) => {
-              console.warn(
-                'An error occurred with cache management for an image',
-                sourceProp.uri,
-                err
-              );
-              mounted && setSource(sourceProp);
-            })
-        : mounted && setSource(sourceProp);
+      if (sourceProp.uri) {
+        fetch(sourceProp.uri)
+          .then((response) => {
+            if (response?.status == 200) {
```

Comment:
could this result in cases, where we have no image now compared with before? do we need to see more status as ok and fine here?

Reply:
nice that you have checked the code and what you found out. love it, let's do it the way you suggested.